toolkit/scripts/pkggen.mk

```diff
@@ -141,7 +141,8 @@ $(cached_file): $(graph_file) $(go-graphpkgfetcher) $(chroot_worker) $(pkggen_lo
 		$(logging_command) \
 		--input-summary-file=$(PACKAGE_CACHE_SUMMARY) \
 		--output-summary-file=$(PKGBUILD_DIR)/graph_external_deps.json \
-		--output=$(cached_file) && \
+		--output=$(cached_file) \
+		--unresolved-pkg-file=$(PKGBUILD_DIR)/unresolved_packages.txt && \
```

Comment:
@0xba1a, could you add more details about the purpose of this change? I believe it is normal for the package fetcher to not find some dependencies, as they may be provided by the packages that are going to be built. Saving that set might be confusing, since the unresolved dependencies can end up being completely resolved by the package build stage.

In addition to that, this tool already has a log file it outputs to. If you're missing some crucial info, why not just add another log command (files will contain everything above "trace" level) instead?